02_julia_intro/02_julia_intro.jl

```diff
@@ -508,10 +545,16 @@ julia> arr
  1
  2
 ```
-Did you understand what happened? At first, we defined and array. Then, we applied the pop!() function, which, as the name suggests, pops the last element of the array. But notice that when we call our *arr* variable to see what it is storing, now the number 3 is gone. This is what functions with a bang do and what we mean with modifying *in-place*. Try to follow this convention whenever you define a function that will modify other objects in-place!
+Did you understand what happened? At first, we defined an array. Then, we applied the pop!() function, which, as the name suggests, pops the last element of the array. But notice that when we call our *arr* variable to see what it is storing, now the number 3 is gone. This is what functions with a bang do and what we mean with modifying *in-place*. Try to follow this convention whenever you define a function that will modify other objects in-place!
 
-Sometimes, you will be in a situation where you may need to use some function, bit you really aren't interested in giving it a special name and storing it. For this kind of situations, an *anonymous* or *lambda* function is what you may need. Typically anonymous functions will be used as arguments to *higher-order functions*. This is just a fancy name to functions that accept other functions as arguments, that is what makes them of higher-order. One of the most ubiquitous example of these type of functions is the ```map()``` function. You can think of this function as a way to broadcast **any** function over a collection.
-Anonymous functions are created using the arrow ```->``` syntax. At the left-hand side of the arrow, you must specify what the arguments of the function will be and their name. At the right side of the arrow, you write the recipe of the things to do with this arguments. Let's use an anonymous function to define a not-anonymous function, just to illustrate the point.
+Sometimes, you will be in a situation where you may need to use some function, but you really aren't interested in giving it a special name and storing it.
+For these kinds of situations, an *anonymous* or *lambda* function is what you may need.
+Typically anonymous functions will be used as arguments to *higher-order functions*. This is just a fancy name to functions that accept other functions as arguments, that is what makes them of higher-order.
+We can create an anonymous function using the ```map()``` keyword. 
+You can think of this function as a way to broadcast **any** function over a collection.
+Anonymous functions are created using the arrow ```->``` syntax. At the left-hand side of the arrow, you must specify what the arguments of the function will be and their name.
+At the right side of the arrow, you write the recipe of the things to do with these arguments.
+Let's use an anonymous function to define a not-anonymous function, just to illustrate the point.
```

Comment:
Below: 

> You can think about what we done here as if $f$ were a variable that is storing some function.

You can think about what we did as if $f$ were a variable that is storing some function.

Reply:
> You can think of them as an array

You can think of them as arrays

Reply:
> In this example the keys will be strings and the values integers.

In this example, the keys will be strings and the values, integers.

Reply:
> Notice that the ways of adding and modifying an element are identical.
> That is because keys of a dictionary can never be repeated, each key must be unique and once added they can't be modified.

Notice that the ways of adding and modifying a value are identical.
That is because keys of a dictionary can never be repeated or modified. Since each key is unique, assigning a new value for a key overrides the previous one.

Reply:
> When we want to modify a key's value.

We do something similar to modify a key's value:

Reply:
Add spaces before and after the "equal" sign in the code

julia> languages["Python"]=1991 > julia> languages["Python"] = 1991